src/test/java/ZarrTest.java

```diff
@@ -0,0 +1,132 @@
+import java.io.File;
```

Comment:
License header please, safe to go ahead with an OME one I think

Reply:
will do, though I see no reason for it to differ from the rest in the repo. Want to propose a package as well?